src/main/java/com/ericsson/ei/frontend/utils/BackendInstancesHandler.java

```diff
@@ -0,0 +1,143 @@
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Component
+public class BackendInstancesHandler {
+    private static final Logger LOG = LoggerFactory.getLogger(BackendInstancesHandler.class);
+
+    @Value("${ei.backend.instances.list.json.content:#{null}}")
+    private String backendInstancesJsonString;
+    private List<BackendInstance> backendInstances = new ArrayList<>();
+
+    @PostConstruct
+    private void init() {
+        parseBackendInstancesStringToBackendInstances();
+    }
+
+    /**
+     * Returns the backendInstances as JsonArray
+     * @return JsonArray
+     */
+    public JsonArray getBackendInstancesAsJsonArray() {
+        JsonArray backendJsonArray = new JsonArray();
+        for (BackendInstance backendInstance : backendInstances) {
+            backendJsonArray.add(backendInstance.getAsJsonObject());
+        }
+        return backendJsonArray;
+    }
+
+    /**
+     * Returns the wanted BackendInstance
+     *
+     * @param backendName
+     * @return BackendInstance
+     * @throws EiBackendInstancesException
+     */
+    public BackendInstance getBackendInstance(String backendName) throws EiBackendInstancesException {
+        for(BackendInstance backendInstance: backendInstances) {
+            if(backendInstance.getName().equals(backendName)) {
+                LOG.debug("Returning backend " + backendName + ".");
+                return backendInstance;
+            }
+        }
+
+        throw new EiBackendInstancesException(
+                "EI Backend instance '" + backendName +  "' does not exist in EI Backend Instances list.");
+        }
+
+    /**
+     * Returns the default backend if set, otherwise it will return
+     * the first instance in the list.
+     *
+     * @return BackendInstance
+     */
+    public BackendInstance getDefaultBackendInstance() {
+        for(BackendInstance backendInstance: backendInstances) {
+            if(backendInstance.isDefaultBackend()) {
+                return backendInstance;
+            }
+        }
+
+        return backendInstances.get(0);
+    }
+
+    /**
+     * Sets the backendInstances list, useful when mocking for tests
+     *
+     * @param backendInstances
+     */
+    public void setBackendInstances(JsonArray backendInstances) {
+        cleanBackendInstances();
+        backendInstancesJsonString = backendInstances.toString();
+        parseBackendInstancesStringToBackendInstances();
+    }
+
+    private void parseBackendInstancesStringToBackendInstances() {
+        try {
+            if(backendInstancesJsonString == null && backendInstancesJsonString.isEmpty()) {
+                LOG.debug("No backend instances provided.");
+                return;
+            }
+
+            JSONArray backendInstancesJsonArray = new JSONArray(backendInstancesJsonString);
+            for(Object jsonElement: backendInstancesJsonArray) {
+                JSONObject backendInstanceJsonObject = (JSONObject)jsonElement;
```

Comment:
Is it not possible to make it a JsonObject in the for statement and not first an Object?

Reply:
Nope sorry. doesnt work :(